dateparser/date.py

```diff
@@ -94,7 +95,7 @@ def sanitize_date(date_string):
     date_string = re.sub(r'\b([ap])(\.)?m(\.)?\b', r'\1m', date_string, flags=re.DOTALL | re.I)
     date_string = re.sub(r'^.*?on:\s+(.*)', r'\1', date_string)
 
-    date_string = re.sub(ur'|'.join(APOSTROPHE_LOOK_ALIKE_CHARS), u"'", date_string)
+    date_string = re.sub(r'|'.join(APOSTROPHE_LOOK_ALIKE_CHARS), u"'", date_string)
```

Comment:
Python 3 has dropped support for `ur` prefix. That's why changed it to raw string. But since, there aren't any back slashes, I'd change the prefix to `u`.i.e. unicode.